src/ui/components/Library/Testsuites.module.css

```diff
@@ -29,6 +29,7 @@
   gap: 0.5rem;
   overflow: hidden;
   border-radius: 0.375rem;
+  padding: 4px;
```

Comment:
`px` instead of `rem`?

Reply:
Yup! We have a Notion page that discusses [when to use rem versus px](https://www.notion.so/replayio/When-to-use-rem-vs-px-f5fbb65a4cad4e0c936ee92581074bb8?pvs=4):

* rem if you want it to resize (think of it like an SVG, designed to scale)
* px if you don't (think of it like a PNG, not designed to scale)

In these three visuals, I am resizing the text but I want the padding to stay the same:
<img width="1065" alt="image" src="https://github.com/replayio/devtools/assets/9154902/40c325ee-2b77-4075-a280-62c058079d71">

And that's why px is the right call! I could allow the capsule padding to get larger with the text, but it has more downsides than benefits.